### DISCUSSION TITLE
winget-cli as a (single) static binary

### DISCUSSION THREAD

**Discussion by thorstenbutz** [microsoft/winget-cli#3603] [General] [answered]
While chatting with @denelon, Demitrius mentioned that someone created a static compiled / single-exe-file version of WingetCLI from the sources. Likely one of the maintainers of this very repo. 
Just out of curiosity, for research purposes: can someone point to that fork - assuming that it is public (I could not find it yet).

**Comment by denelon** [comment, marked as answer]
@jedieaston has one over at https://github.com/jedieaston/winget-build

**Reply by thorstenbutz** [reply]
Great, thanks for the hint!
Thorsten
